src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class Rectangle implements Figure {
```

Comment:
The class 'Rectangle' should not be named with a verb. Consider renaming it to something more appropriate that describes what it is or does.